extensions/vertx-http/dev-console-spi/src/main/java/io/quarkus/devconsole/spi/DevConsoleRouteBuildItem.java

```diff
@@ -14,7 +14,7 @@
 /**
  * A route for handling requests in the dev console.
  * <p>
- * Routes are registered under /q/dev-v1/{groupId}.{artifactId}/
+ * Routes are registered under /q/dev-ui/{groupId}.{artifactId}/
```

Comment:
Actually, this one shouldn't be changed as this build item is about the deprecated dev-v1.

Reply:
88616ceaa6a3bdb80d861807ad6fb7ef097160f1 closes.